docs/create-next-app.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Create Next.js apps in one command with create-next-app.
+---
+
+# Create Next App
+
+The easiest way to get started with Next.js is by using `create-next-app`. This simple CLI tool enables you to quickly start building a new Next.js application, with everything set up for you. You can create a new app using the default Next.js template, or by using one of the [official Next.js examples](https://github.com/vercel/next.js/tree/canary/examples). To get started, use the following command:
+
+```bash
+npx create-next-app
+# or
+yarn create next-app
+```
+
+To create a new app in a specific folder, you can send a name as an argument. For example, the following command will create a new Next.js app called `blog-app` in a folder with the same name:
+
+```bash
+npx create-next-app blog-app
+# or
+yarn create next-app blog-app
+```
```

Comment:
I don't think we need this since we prompt you for it.

Reply:
Fixed!